helm/secdash/dags/_utils.py

```diff
@@ -56,3 +61,37 @@ def split_array(original_array, num_subarrays):
         subarrays[i].append(original_array[num_subarrays * subarray_size + i])
 
     return subarrays
+
+
+def get_email_hash(email):
+    return hashlib.md5(email.lower().encode('utf-8')).hexdigest()
+
+
+def fetch_unique_emails(mongo_conn_id):
```

Comment:
Let's move this to `update_mailchimp_list.py`. It doesn't belong as a method of `MailchimpManager`, but it can stay in the same file since the function isn't generic and shares the same context as `MailchimpManager`.